packages/twenty-front/src/modules/object-record/object-options-dropdown/components/ObjectOptionsDropdownMenuContent.tsx

```diff
@@ -117,15 +118,20 @@ export const ObjectOptionsDropdownMenuContent = () => {
           contextualText={`${visibleBoardFields.length} shown`}
           hasSubMenu
         />
-        {(viewType === ViewType.Kanban || isViewGroupEnabled) && (
-          <MenuItem
-            onClick={() => onContentChange('recordGroups')}
-            LeftIcon={IconLayoutList}
-            text="Group by"
-            contextualText={recordGroupFieldMetadata?.label}
-            hasSubMenu
-          />
-        )}
+        {(viewType === ViewType.Kanban || isViewGroupEnabled) &&
+          currentView?.key !== 'INDEX' && (
```

Comment:
Why do we have this change ? I can't see the group by option from the table view, had to remove it to be able to create a view group

Reply:
Oh I see, it is to disable the option from the main view right ? 
I feel this may prevent users from discovering the feature. Instead I would create a new view when selecting group by. wdyt @Bonapara ?

Reply:
@Bonapara not sure you have seen this comment, would just like to double check that it is the intended behaviour

Reply:
@ijreilly It was actually decided on Discord with @charlesBochet to go that way for now